pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
 
-test_aspects:
+input_aspects:
   -  # (e.g. HTTP Messages, DOM Tree, CSS Styling, Accessibility Tree, Language, etc.,)
 
 authors:
   -  # (full names as found in `contributors` property in  `package.json` - if not yet listed, please have authors added to the list)
 ---
 
-## Test Procedure
-
-### Applicability
+## Applicability
 
 The rule applies to any (??) element ...
 
-### Expectation (1)
+## Expectation (1)
```

Comment:
What did we agree on here? 
Does expectations still only have to be numbered, if there are multiple of them?

Reply:
Only number them when there is more than one expectation.